docs/architecture/cloud-native/Database-per-microservice.md

```diff
@@ -0,0 +1,148 @@
+---
+title: Database-per-microservice
+description: Contrast data storage in monolithic and cloud-native applications
+author: robvet
+ms.date: 12/23/2019
+---
+# Database-per-microservice
```

Comment:
Can you remove multiple links of `materialized view pattern` and `materialized views` on this page? Felt too many links that can distract the reader.

Reply:
Done!  Good catch.